docs/changelog.md

```diff
@@ -70,9 +70,10 @@ permalink: /changelog/
 * **concept-storage**
   * ⚠️ **This is a breaking API change!** for non-component implementations of `HistoryStorage`.
   * `HistoryStorage` got new APIs: `deleteEverything`, `deleteVisitsSince`, `deleteVisitsBetween`, `deleteVisitsFor`, `prune` and `runMaintenance`.
+  * Added `BookmarksStorage` for handling the saving, searching, and management of browser bookmarks.
 
 * **browser-storage-sync**, **browser-storage-memory**
-  * Implementations of `concept-storage`/`HistoryStorage` expose the newly added APIs.
+  * Implementations of `concept-storage`/`HistoryStorage` and `concept-storage`/`BookmarksStorage` expose the newly added APIs.
```

Comment:
This says that browser-storage-memory also got the implementation, which isn't the case.

Reply:
Updated